### PR TITLE
Makefile: add uninstall

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
LGTM, thought we'd need to remove the completions too, but they're apparently not installed with `make install`

File: Makefile [hunk 1]
```diff
@@ -82,6 +82,11 @@ install:
 	install -D -m0755 buildah $(DESTDIR)/$(BINDIR)/buildah
 	$(MAKE) -C docs install
 
+.PHONY: uninstall
+uninstall:
+	rm -f $(DESTDIR)/$(BINDIR)/buildah
+	rm -f $(PREFIX)/share/man/man1/buildah*.1
```
Comment (rhatdan) [inline Makefile]: What about completions and other stuff that you installed?

Comment (TomSweeneyRedHat) [reply Makefile]: We only install these two with `make install`.  Should this remove everything that we could potentially install other commands like `make install.completions`?  @rhatdan should we add completions to the standard `make install`?

Comment (rhatdan) [reply Makefile]: No I don't think so, but the reason for a make uninstall is to remove cruft from your system, I guess although it is seldom done.
If we are going to add a make uninstall, we should have either a make uninstall.completions... or just have make uninstall all of the stuff that the Makefile could have added.

Comment (q384566678) [reply Makefile]: updated, Not sure if it is correct.

File: Makefile [hunk 2]
```diff
@@ -82,6 +82,14 @@ install:
 	install -D -m0755 buildah $(DESTDIR)/$(BINDIR)/buildah
 	$(MAKE) -C docs install
 
+.PHONY: uninstall
+uninstall:
+	rm -f $(DESTDIR)/$(BINDIR)/buildah
+	rm -f $(DESTDIR)/$(BINDIR)/runc
```
Comment (rhatdan) [inline Makefile]: runc did not come from this package.

File: Makefile [hunk 3]
```diff
@@ -82,6 +82,14 @@ install:
...
+	rm -f $(PREFIX)/share/man/man1/buildah*.1
+	rm -f $(DESTDIR)/${BASHINSTALLDIR}/buildah
+	rm -r /opt/cni/bin
```
Comment (rhatdan) [inline Makefile]: I would also leave this alone, runc and /opt/cni/bin can be used by other packages.

**Comment by q384566678** [comment]
updated.

**Comment by TomSweeneyRedHat** [comment]
LGTM and happy green test buttons.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**